Solutions/GitHub/Parsers/GithubSecretScanningData.txt

```diff
@@ -0,0 +1,34 @@
+// GitHub Enterprise repository vulnerability Data Parser
+// Last Updated Date: June 15, 2022
+//
+//This parser parses GitHub Enterprise secret scanning events and extract the information from their various components. It is assumed that you are using officially supported Github webhook connector (installed from Content hub)
+//
+// Parser Notes:
+// 1. This parser pulls the logs those are collected into a custom log table entitled githubscanaudit_CL 
+
+githubscanaudit_CL 
+|where action_s in ('created', 'resolved' ,'reopened')
+|extend EventType='dependabotAlert-SecretScanningAlert'
+|extend alert = todynamic(alert_s),
+ organization = todynamic(organization_s),
+ repository = todynamic(repository_s),
+sender = todynamic(sender_s) 
+|extend   
+        alertSecretType = alert.secret_type,
+        alertnumber = alert.number,
+        alertresolution = alert.resolution,
+        alertresolvedby = alert.resolved_by,
+        alertresolvedat = alert.resolved_at,
+        repositoryfullname = repository.full_name,
+        repositoryOwnerlogin = repository.owner.login,
+        repositoryurl = repository.url,       
+        senderlogin = sender.login,
+        sendertype = sender.type
+|where alertnumber == 191 
+|project TimeGenerated, EventType,  action_s, alertSecretType, alertnumber,alertresolution, alertresolvedby,alertresolvedat, repositoryfullname,repositoryOwnerlogin, repositoryurl, senderlogin, sendertype 
```

Comment:
Rename the action_s field as well. This is applicable for all three parsers.

Reply:
done